.changesets/breaking_lb_next_apollo_otel_default.md

```diff
@@ -0,0 +1,7 @@
+### feat: default to Apollo reporting over OTel ([PR #5741](https://github.com/apollographql/router/pull/5741))
+
+BREAKING CHANGE: This change will cause traces in Apollo Studio to appear differently
```

Comment:
Should we include the config snippet to turn this back off?

Reply:
great idea, i don't know the answer. it's possible that by the time 2.0 is GA, the option will be removed entirely, so i'm not going to worry about it right now